.travis.yml

```diff
@@ -159,9 +159,8 @@ script:
   - export CMAKE_ARGS="-DLUA_LIBRARY=${LUALIBRARY} -DLUA_INCLUDE_DIR=${LUAINCLUDE} -D OVERRIDE_VERSION=$AWESOME_VERSION -DSTRICT_TESTS=true"
   - |
     if [ "$DO_COVERAGE" = "codecov" ]; then
-      export CXXFLAGS="-fprofile-arcs -ftest-coverage"
       export CFLAGS="-fprofile-arcs -ftest-coverage"
-      export LDFLAFS="-fprofile-arcs -ftest-coverage"
+      export LDFLAGS="-fprofile-arcs"
```

Comment:
Uhm, why? (And I think that should be part of the commit message)

For `$CXXFLAGS` I guess the reason is that we have no C++ code.
For `$LDFLAGS` I don't know. If I had to guess, I would say that this is not actually a valid linker flag.

Please improve the commit message.